cpp/arcticdb/CMakeLists.txt

```diff
@@ -27,9 +28,17 @@ find_package(Gflags REQUIRED)
 find_package(LZ4 REQUIRED)
 find_package(prometheus-cpp CONFIG REQUIRED)
 find_package(Threads REQUIRED)
-find_package(unofficial-pcre REQUIRED)
+if(NOT ARCTICDB_USING_CONDA)
+    find_package(unofficial-pcre REQUIRED)
+else()
+    find_package(PCRE REQUIRED)
+endif()
```

Comment:
Can this be moved bellow?

Reply:
sure! not in love with any change I made, whatever makes is nice is fine